Source/WebCore/rendering/RenderText.cpp

```diff
@@ -1329,6 +1330,73 @@ UChar RenderText::previousCharacter() const
     return previousString[previousString.length() - 1];
 }
 
+static String convertToFullSizeKana(const String& string)
+{
+    // https://www.w3.org/TR/css-text-3/#small-kana
+    static constexpr std::pair<uint16_t, uint16_t> kanasMap[] = {
+        { 0x3041, 0x3042 },
+        { 0x3043, 0x3044 },
+        { 0x3045, 0x3046 },
+        { 0x3047, 0x3048 },
+        { 0x3049, 0x304A },
+        { 0x3063, 0x3064 },
+        { 0x3083, 0x3084 },
+        { 0x3085, 0x3086 },
+        { 0x3087, 0x3088 },
+        { 0x308E, 0x308F },
+        { 0x3095, 0x304B },
+        { 0x3096, 0x3051 },
+        { 0x30A1, 0x30A2 },
+        { 0x30A3, 0x30A4 },
+        { 0x30A5, 0x30A6 },
+        { 0x30A7, 0x30A8 },
+        { 0x30A9, 0x30AA },
+        { 0x30C3, 0x30C4 },
+        { 0x30E3, 0x30E4 },
+        { 0x30E5, 0x30E6 },
+        { 0x30E7, 0x30E8 },
+        { 0x30EE, 0x30EF },
+        { 0x30F5, 0x30AB },
+        { 0x30F6, 0x30B1 },
+        { 0x31F0, 0x30AF },
+        { 0x31F1, 0x30B7 },
+        { 0x31F2, 0x30B9 },
+        { 0x31F3, 0x30C8 },
+        { 0x31F4, 0x30CC },
+        { 0x31F5, 0x30CF },
+        { 0x31F6, 0x30D2 },
+        { 0x31F7, 0x30D5 },
+        { 0x31F8, 0x30D8 },
+        { 0x31F9, 0x30DB },
+        { 0x31FA, 0x30E0 },
+        { 0x31FB, 0x30E9 },
+        { 0x31FC, 0x30EA },
+        { 0x31FD, 0x30EB },
+        { 0x31FE, 0x30EC },
+        { 0x31FF, 0x30ED },
+        { 0xFF67, 0xFF71 },
+        { 0xFF68, 0xFF72 },
+        { 0xFF69, 0xFF73 },
+        { 0xFF6A, 0xFF74 },
+        { 0xFF6B, 0xFF75 },
+        { 0xFF6C, 0xFF94 },
+        { 0xFF6D, 0xFF95 },
+        { 0xFF6E, 0xFF96 },
+        { 0xFF6F, 0xFF82 }
+    };
+    constexpr auto sortedMap = SortedArrayMap { kanasMap };
```

Comment:
Not sure why the one above is `static constexpr` and this is just `constexpr`. I believe they should both be the same since there are no different considerations for the two. I also suggest omitting `auto` and `=` and just writing `static constexpr SortedArrayMap sortedMap { kanasMap };`.

Reply:
Changed to use `static`.